### ISSUE TITLE
The argument stepNum of the startTour method is not working

### ISSUE THREAD

**Issue by gianluca78** [linkedin/hopscotch#129]
Hi everyone,
I think there is a bug in the startTour function (version 0.2.2). The parameter stepNum is always ignored and the tour starts every time from the first step.

I have identified the problem in lines 1724-1727:

      if (!utils.documentIsReady()) {
        waitingToStart = true;
        return this;
      }

This code doesn't keep track of the stepNum parameter but I don't know how to fix it.

Thank you very much.

**Comment by kate2753** [comment]
Thank you for reporting this issue! It has the same root cause as in #130 It'll be fixed in 0.3.0 release. I'm closing this ticket for now. For further updates keep an eye on #130

**Comment by StoyanPenev** [comment]
Hopscotch actually looses the stepNum argument if the document.readyState !== 'complete' when the hopscotch.startTour(tour, stepNum) is called.
From hopscotch.js startTour(tour, stepNum):
```js
if (!utils.documentIsReady()) {
      waitingToStart = true;
      return this;
    }
```
and then we go in this handler:
```js
winLoadHandler = function() {
  if (waitingToStart) {
    winHopscotch.startTour(); //no step here?
  }
};
```
A fix for this is not $(document).ready(callback) because it executes the callback even if document.readyState === 'interactive' which is not what hopscotch is waiting looking for, the following fixed it for me.
```js
function ready(callback) {
    document.onreadystatechange = function() {
        if (document.readyState=='complete') callback();
    };
}
// Fixes hopscotch start tour from specific step issue, @see hopscotch.js startTour(tour, stepNum) and utils.documentIsReady().
ready(function(){
    //your code here
    hopscotch.startTour(theTour, startFromStep);
});
```

I think this needs to be addressed in the source as  its quite a basic feature.
Hope this helps you meanwhile :)
